### ISSUE TITLE
TileRequest: stylecloud tile

### ISSUE THREAD

**Issue by breck7** [breck7/ohayo#27]
There's a cool new python word cloud package. Would be neat to implement this an ohayo tile(s) to replace current wordcloud.

https://github.com/minimaxir/stylecloud

**Comment by ghost** [comment]
@breck7 Can I work on this?

**Comment by breck7** [comment]
@udaykiranreddykondreddy sure! Let me know if you are able to successfully setup your environment. Are you on windows, Mac, or Linux?

**Comment by ghost** [comment]
@breck7 Yes I'm good with the environment setup now. I'm on both Windows and Linux and installed this on both machines.
